tests/manage/pv_services/pvc_snapshot/test_pvc_snapshot_when_snapshotclass_deleted.py

```diff
@@ -0,0 +1,148 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier3,
+    skipif_ocs_version,
+    skipif_ocp_version,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources import pvc, ocs
+from ocs_ci.helpers import helpers
+from ocs_ci.utility import templating
+
+log = logging.getLogger(__name__)
+
+
+@tier3
+@skipif_ocs_version("<4.7")
+@skipif_ocp_version("<4.7")
+@pytest.mark.polarion_id("OCS-2638")
+class TestPvcSnapshotWhenSnapshotClassDeleted(ManageTest):
+    """
+    Tests to verify PVC snapshot feature when snapshotclass deleted
+    """
+
+    @pytest.fixture(autouse=True)
+    def base_setup(self, interface_iterate, pvc_factory, pod_factory):
+        """
+        Create resources for the test
+
+        Args:
+            interface_iterate: A fixture to iterate over interfaces
+            pvc_factory: A fixture to create new pvc
+            pod_factory: A fixture to create new pod
+
+        """
+
+        # Create resources for the test
+        self.interface = interface_iterate
+
+        self.pvc_obj = pvc_factory(
+            interface=self.interface, size=5, status=constants.STATUS_BOUND
+        )
+
+        self.pod_object = pod_factory(
+            interface=self.interface, pvc=self.pvc_obj, status=constants.STATUS_RUNNING
+        )
+
+    def create_snapshotclass(self, interface):
```

Comment:
It may not be re-used since we don't recommend creating our own snapshot class. 

Here in this test case, we are deleting the snapshot class which used to create a snapshot so we are creating our own snapshot class not using/deleting the default one.